src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue

```diff
@@ -145,6 +145,12 @@ export default {
 				return p1inCall ? -1 : 1
 			}
 
+			const p1HandRaised = this.$store.getters.isParticipantRaisedHand(participant1.sessionId)
+			const p2HandRaised = this.$store.getters.isParticipantRaisedHand(participant2.sessionId)
+			if (p1HandRaised !== p2HandRaised) {
+				return p1HandRaised ? -1 : 1
```

Comment:
Makes sense. the problem is if sort by that I would assume it's by order of the raising and not the remaining things?
But we don't have a timestamp or something :/

Reply:
right. in theory we can add a timestamp but it wouldn't be authoritative as it would be based on whenever the signal/event is received, so everyone might see something different

unless we send the timestamp from the sender and use that instead (would be easy to cheat but don't think we should care)

Reply:
> unless we send the timestamp from the sender and use that instead (would be easy to cheat but don't think we should care)

sounds good enough. Is that easily doable?

Reply:
yeah, I'll have a try.

there is another issue to look into: it seems reactivity is not fully working from inside this sort function, probably because accessing the hand state is "too remote".
I had cases where the list didn't re-sort automatically, it only did after a full refresh

Reply:
as discussed in chat: I'll remove the sorting for now and we can investigate separately